newsroom/templates/search_tips_advanced.html

```diff
@@ -43,10 +43,21 @@
     <hr class="dashed" />
     <div class="advanced-search__content-block advanced-search__content-block--no-background">
         <p class="advanced-search__content-block-text advanced-search__content-block-text--bold">
-            Wildcard and Quotation Marks are accepted in the three search input fields on Advanced Search
+            Wildcards, Quotation Marks, and proximity search are accepted in the three search input fields on the Advanced Search form, if desired:
         </p>
         <ul>
-            <li>You can use Wildcard operator ? to replace a single character or * to replace zero or more characters and Quotation Marks "" to search for an exact phrase</li>
+            <li>Use Wildcard operator <span class="advanced-search__content-block-text--mono">?</span> to replace a single character or <span class="advanced-search__content-block-text--mono">*</span> to replace zero or more characters</li>
```

Comment:
just following fritz's design

Reply:
I'm not questioning the design, rather the markup

Reply:
but ill still need loooong className like u say :D

Reply:
why would you need it?

Reply:
What do you suggest Petr, rewrite CSS to classnames shorter?

Reply:
I would use eg `code` instead of `span` and do the styling for `code` tag, if needed based on some high level `search-info-page` class

Reply:
to keep it somehow semantic

Reply:
imo this page is not a component so I wouldn't use bem for it

Reply:
I see. Yep, since the document is really simple we can use semantic markup here and style tags, given that CSS is namespaced and not made global. @dzonidoo can you change it accordingly then?